runner/status.go

```diff
@@ -151,3 +151,12 @@ func PreparingStatus(runID RunID) (r RunStatus) {
 	r.State = PREPARING
 	return r
 }
+
+// This is for overall runner status, just 'initialized' status for now.
+type ServiceStatus struct {
```

Comment:
do we want to have a struct with member variable or int type and use enum (iota) values?